### PR TITLE
Add additional scrrenshots

### PR DESCRIPTION
**Pull request by Samantha-Stevens**
#### What does this PR do?

Adds additional screenshots of website to show functionality.

#### Where should the reviewer start?

At the images section

#### How should this be manually tested?

Check that the links work.

#### Any background context you want to provide?

Second set of images added to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,11 +61,15 @@ Project spec & rubric can be found [here](https://frontend.turing.io/projects/mo
 [Hang in there group project](https://joanafbrito.github.io/hang-in-there-boilerplate/)
 
 ## Images/gifs:
```
Comment (williamphelps13) [inline README.md]: We should remove gif because we don't have any

File: README.md [hunk 2]
```diff
@@ -61,11 +61,15 @@ Project spec & rubric can be found [here](https://frontend.turing.io/projects/mo
...
-![Main page Poster](https://user-images.githubusercontent.com/82066350/119276133-fb13ca00-bbcd-11eb-9071-83c7e934e71c.png)
+![Main page poster](https://user-images.githubusercontent.com/82066350/119276133-fb13ca00-bbcd-11eb-9071-83c7e934e71c.png)
```
Comment (williamphelps13) [inline README.md]: I noticed that this photos include part of my browser window including the tabs. Can you crop these to remove that?

**Review by williamphelps13 — CHANGES_REQUESTED** [review]
Good additions! Unfortunately I think the screenshot cropping needs to occur before merge.

File: README.md (hunk 2 above)
Comment (Samantha-Stevens) [reply README.md]: William, that was a good  catch on the images, I have re-cropped the screenshots.

File: README.md [hunk 3]
```diff
@@ -60,16 +60,16 @@ Project spec & rubric can be found [here](https://frontend.turing.io/projects/mo
...
+![Create your own poster main page](https://user-images.githubusercontent.com/79541611/119277866-0ed02800-bbf0-11eb-9d2a-a6ced8910c68.png)
 
-![Your created poster](https://user-images.githubusercontent.com/79541611/119277065-82236b00-bbeb-11eb-858b-36555de1cc6c.png")
+![Your created poster](https://user-images.githubusercontent.com/79541611/119277912-48089800-bbf0-11eb-9013-e8025d85d4a4.png)
 
-![Saved posters grid view](https://user-images.githubusercontent.com/79541611/119277225-65d3fe00-bbec-11eb-96c1-bd07fca0c2e0.png)
+![Saved posters grid view](https://user-images.githubusercontent.com/79541611/119277957-75eddc80-bbf0-11eb-8771-07500dead766.png)
 
-![Shows what you deleted from saved](https://user-images.githubusercontent.com/79541611/119277151-ed6d3d00-bbeb-11eb-9233-ed20ff9031af.png)
+![Shows what you deleted from saved](https://user-images.githubusercontent.com/79541611/119277986-a2095d80-bbf0-11eb-84e1-e53ec4cf7630.png)
```
Comment (williamphelps13) [inline README.md:72]: Looks great! Pictures are beautifully cropped!

**Pull request merged**